Calculator/Calculator/Model/ExpressionParser.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  ExpressionParser.swift
+//  Calculator
+//
+//  Created by kangkyungmin on 2023/06/07.
+//
+
+enum ExpressionParser {
+    static func parse(from input: String) -> Formula {
+        var operands = CalculatorItemQueue<Double>()
+        var operators = CalculatorItemQueue<Operator>()
+        
+        componentsByOperators(from: input).compactMap { Double($0) }.forEach { element in operands.enqueue(element) }
+
+        input.compactMap { Operator(rawValue: $0) }.forEach { element in operators.enqueue(element) }
```

Comment:
1. 후행 클로저
```suggestion
        componentsByOperators(from: input).compactMap { Double($0) }.forEach { element in 
            operands.enqueue(element) 
        }

        input.compactMap { Operator(rawValue: $0) }.forEach { element in 
            operators.enqueue(element) 
        }
```

[후행클로저](https://docs.swift.org/swift-book/documentation/the-swift-programming-language/closures/#Trailing-Closures)의 컨벤션은 보통 이렇게 씁니다.

2. 단축 인자
```suggestion
        componentsByOperators(from: input).compactMap { Double($0) }.forEach { operands.enqueue($0) }

        input.compactMap { Operator(rawValue: $0) }.forEach { operators.enqueue($0) }
```

`compactMap`에서는 `단축 인자`를 사용하셨는데 `forEach`에서는 사용 안 하신 이유가 궁금합니다!

Reply:
element를 enqueue해주는다는 느낌을 주고 싶었습니다
단축인자로 통일하는것이 좋다고 판단하여 고쳤습니다😊